src/toolkits/style_transfer/style_transfer_data_iterator.hpp

```diff
@@ -0,0 +1,96 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef __TOOLKITS_STYLE_TRANSFER_H_
+#define __TOOLKITS_STYLE_TRANSFER_H_
+
+#include <functional>
+#include <map>
+#include <memory>
```

Comment:
I don't see any of these being used. You do however use `string` and `random`

Reply:
Copied from OD. I'll go through and clean up the includes.